### PR TITLE
Agent setup new page

### PR DESCRIPTION
**Pull request by Rookeret73**

### CODE REVIEW

File: docs/agent-setup.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+The Rookout agent has HTTPS proxy support for advanced network configurations.  
+As most production systems do have automated means for proxy detection, you should configure it statically.  
+
+This can be done in the Agent’s installation:
+```bash
+$ setup.sh --token=[Your Rookout Token] --https-proxy=[Your Proxy Server]
+```
+
+Or by setting the environment variable HTTPS_PROXY.
```
Comment (LiranLast) [inline docs/agent-setup.md]: @Rookeret73 add an example :)

File: docs/agent-setup.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+### Disable Sending Data
+
+The agent connects to the Rookout Service to receive commands and report telemetry information.
+Data collected from within the application may be sent to Rookout for interactive debugging sessions.
+To prevent sending data to the Rookout Service due to security restrictions, set the environment variable ROOKOUT_SEND_DATA to false.
+
+### Local Rule Targets
+
+As the Agent runs within your network, it allows you to direct the collected data into data sinks.
+For example, debug message can be sent to Elasticsearch or Splunk clusters.
```
Comment (LiranLast) [inline docs/agent-setup.md]: messages

File: docs/agent-setup.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+```bash
+$ export ROOKOUT_TOKEN=[Your Rookout Token]
+$ curl -fs https://get.rookout.com | bash
+```
+
+Alternatively, you can download the script directly from the following link: https://get.rookout.com  
+And run it using the following commands:
+```bash
+$ setup.sh --token=[Your Rookout Token]
```
Comment (LiranLast) [inline docs/agent-setup.md]: Bash should include how to download it

File: docs/agent-setup.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+### 2. Using Kubernetes and Helm
+
+If you are running on Kubernetes, we recommend using our provided helm chart available [here](https://github.com/helm/charts/tree/master/stable/rookout).
+
+To use it, run the following command:
+
+```bash
+$ helm install --name [Your release name] stable/rookout --set token=[Your Rookout Token]
```
Comment (LiranLast) [inline docs/agent-setup.md]: Add a default release name such as `rookout-agent`

Comment (Rookeret73) [reply docs/agent-setup.md]: It's not the rookout release name.
It's the user's helm chart release name.

**Comment by Rookeret73** [comment]
@LiranLast applied fixes and pushed.

File: docs/agent-setup.md (hunk 1 above)
Comment (Rookeret73) [reply docs/agent-setup.md]: Done

File: docs/agent-setup.md (hunk 2 above)
Comment (Rookeret73) [reply docs/agent-setup.md]: fixed

File: docs/agent-setup.md (hunk 4 above)
Comment (LiranLast) [reply docs/agent-setup.md]: @Rookeret73 which should have a default name. No reason the user should have to think about it

File: docs/agent-setup.md (hunk 3 above)
Comment (LiranLast) [reply docs/agent-setup.md]: @Rookeret73 that won't work. curl writes to stdout rather then file

**Pull request merged**